kotlin/export-tiles/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context=".MainActivity">
+  <!-- MapViews -->
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/mapPreviewLayout"
+    android:layout_width="0dp"
+    android:layout_height="0dp"
+    android:layout_marginBottom="0dp"
+    android:layout_marginEnd="0dp"
+    android:layout_marginTop="0dp"
+    tools:layout_constraintTop_creator="1"
+    tools:layout_constraintLeft_creator="1"
+    app:layout_constraintLeft_toLeftOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    app:layout_constraintRight_toRightOf="parent"
+    app:layout_constraintBottom_toBottomOf="@+id/mapView">
+
+    <View
+      android:id="@+id/dimBackground"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/dimBackground"
+      app:layout_constraintRight_toLeftOf="@+id/closeButton"
+      tools:layout_constraintTop_creator="1"
+      tools:layout_constraintRight_creator="1"
+      tools:layout_constraintBottom_creator="1"
+      app:layout_constraintBottom_toBottomOf="@+id/previewMapView"
+      app:layout_constraintTop_toBottomOf="@+id/previewMapView"
+      tools:layout_constraintLeft_creator="1"
+      app:layout_constraintLeft_toLeftOf="@+id/closeButton" />
+
+    <TextView
+      android:id="@+id/preview_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginLeft="8dp"
+      android:layout_marginTop="8dp"
+      android:layout_marginRight="8dp"
+      android:layout_marginBottom="8dp"
+      android:fontFamily="sans-serif-medium"
+      android:text="@string/preview_text"
+      android:textColor="@color/colorPrimary"
+      android:textSize="26sp"
+      app:layout_constraintBottom_toTopOf="@+id/previewMapView"
+      app:layout_constraintHorizontal_bias="0.494"
+      app:layout_constraintLeft_toLeftOf="parent"
+      app:layout_constraintRight_toRightOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      tools:layout_constraintLeft_creator="1"
```

Comment:
Not sure these creator things are needed (I appreciate those may be leftover from my implementation of the java sample), can you try removing them?

Reply:
nice one, I've taken them out and all is well!